### DISCUSSION TITLE
Does hellosystem support iBus or other input systems?

### DISCUSSION THREAD

**Discussion by DavidLiang1129** [helloSystem/hello#501] [Q&A]
Hello, everyone, I come from China. I installed hellosystem 0.8.1 on my Sony VAIO VPCCA16EC laptop some time ago. All the hardware except the camera works well. Thanks to all the contributors for their hard work! However, I found that although it has Chinese support, it still cannot input Chinese, which causes a certain degree of use barriers for Chinese users including me. On Linux system, I have heard of iBus, but I know hellosystem is based on FreeBSD, so I would like to ask, Does hellosystem support iBus or other input system? Thanks!![IMG_20230326_131950.jpg](https://user-images.githubusercontent.com/118408976/237055659-6051becf-2421-4b5d-9d1f-aabc68bed668.jpg)

**Comment by louies0623** [comment]
I'm also a Chinese user, but it's just traditional. I also requested to add iBus several times for this, but the input method is still not added.

**Reply by DavidLiang1129** [reply]
Hello, I am translating an open source game, are you interested in doing traditional Chinese translation? (Sorry, this doesn't seem like the right place to talk about it, but I can't find any other way to reach you.) If you're interested, send me an E-mail at 2188741623@qq.com

**Reply by louies0623** [reply]
我其實沒有很多的時間可以幫忙翻譯，但是我期望你所做出來的遊戲是能對得起你自己的，由初心所出，也能造就人，且幫助別人，“在小事上用心，在大事上也用心”這是從一本書來的，祝你能夠順利。

**Reply by DavidLiang1129** [reply]
謝謝你的祝福，其實我不是這個遊戲的作者，我只是這個遊戲的一名志願中文翻譯，我翻譯的遊戲是一款寵物捕捉類型RPG，類似《寵物小精靈》如果你對這款遊戲感興趣的話，我們可以留下一個聯繫方式。(很抱歉，為了避免廣告嫌疑，我不能在這裡說出遊戲的名字)

**Reply by louies0623** [reply]
謝謝你的邀請，我很抱歉我對這個沒什麼太大的興趣，但是祝你好運

**Reply by DavidLiang1129** [reply]
沒關係，你不需要道歉，每個人都有自己喜歡和不喜歡或是不感興趣的事物，謝謝你的祝福，也祝你能夠幸福美滿，心想事成。

**Comment by probonopd** [comment]
Please see https://github.com/helloSystem/docs/issues/34.

**Reply by DavidLiang1129** [reply]
Thanks!
